pandas/core/indexes/multi.py

```diff
@@ -896,20 +896,24 @@ def _get_level_values(self, level):
         Parameters
         ----------
         level : int level
+        unique : bool
```

Comment:
you need to add a version added tag

Reply:
(done)

Reply:
need a versionadded tag

Reply:
I added it, and then removed it on @jorisvandenbossche 's suggestion because this is an internal method.

Reply:
add the default value in the doc string

Reply:
u can remove the versionadded tag here